src/components/SlateArea/SlateArea.tsx

```diff
@@ -259,10 +258,11 @@ const SlateArea = ({
             className
           )}
           onKeyDown={(event) => {
-            if (event.key === "Tab") {
-              event.preventDefault();
-              Transforms.move(editor, { distance: 2, unit: "line" });
-            }
+            // @todo What was the purpose of this?
```

Comment:
I have a feeling this was done for tanning through the cells in a table.
Would be nice to have comment here why this is necessary.